department-of-veterans-affairs/caseflow
# SeanC/APPEALS-13696: Adding the Ability to Sort Decision Review Queues by Their SSN Columns

Resolves [APPEALS-13696](https://vajira.max.gov/browse/APPEALS-13696)

### Description
Added the functionality to sort the tasks in the business lines' decision review queue table by veteran_ssn.

### Acceptance Criteria
- [x] Tasks are able to be sorted, in ascending numerical order by the veteran ssn
- [x] Tasks are able to be sorted, in descending numerical order by the veteran ssn
- [x] Added new tests for the new functionality
- [x] Fixed any affected tests

### Testing Plan
1. Sign in as BVAISHAW
2. Go to /decision_reviews/vha page
3. Click on the up arrow icon to the right of the Veteran SSN label
4. Check that the URL is "/decision_reviews/vha?tab=in_progress&page=1&sort_by=veteranSsnColumn&order=asc"
5. Check that the rows are now ordered ascending by the Veteran SSN
6. Click on the down arrow icon to the right of the Veteran SSN label
7. Check that the URL is "/decision_reviews/vha?tab=in_progress&page=1&sort_by=veteranSsnColumn&order=desc"
8. Check that the rows are now ordered descending by the Veteran SSN 

 BEFORE|AFTER
 ---|---
<img width="1316" alt="Before" src="https://user-images.githubusercontent.com/110493538/215882955-229ee3d5-56c8-4186-9d06-2148f7e31727.png">|<img width="1307" alt="After" src="https://user-images.githubusercontent.com/110493538/215882985-d2821727-453f-4f3d-a236-8c5287922cd8.png">
